maxim/tests/test_portkey.py

```
@@ -0,0 +1,29 @@
+from maxim.logger import LoggerConfig
+from maxim.logger.portkey import instrument_portkey
+from maxim.logger.openai.client import MaximOpenAIClient, MaximOpenAIAsyncClient
+
+
+logger = Maxim(Config(api_key="a", base_url="https://example.com")).logger(
+    LoggerConfig(id="repo")
+)
+
+
+class TestPortkeyIntegration(unittest.TestCase):
+    def test_instrument_portkey_sync(self):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_instrument_portkey_sync`

Add return type annotation: `None`

(ANN201)